core/src/main/java/org/bptlab/cepta/patterns/StaysInStationPattern.java

```diff
@@ -0,0 +1,67 @@
+package org.bptlab.cepta.patterns;
+
```

Comment:
Maybe pull the dev again? I don't think we need to add this again with this PR. That seems kinda wrong.